Framework/src/runAdvanced.cxx

```diff
@@ -0,0 +1,152 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    runAdvanced.cxx
+/// \author  Piotr Konopka
+///
+/// \brief This is an executable showing a more complicated QC topology.
+///
+/// This is an executable showing a more complicated QC topology. It pretends to spawn 4 separate topologies - 3 of them
+/// consist of some dummy processing chain, a dispatcher and a local QC task. The last one represents the remote
+/// QC servers topology, which has a merger (joining the results from local QC tasks), a checker (checks the result of
+/// the previous) and a different, remote QC task with associated checker. Here they are joined into one big topology
+/// just to present the concept.
```

Comment:
As you have a screenshot of the resulting topology, I would add it here.

Reply:
A link? Where can I upload it so it does not disappear?

Reply:
Aaa, I can just add the image to the repo...